gossip/gossip.go

```diff
@@ -700,9 +700,6 @@ func (g *Gossip) getNextBootstrapAddress() net.Addr {
 		g.resolverIdx %= len(g.resolvers)
 		g.resolversTried[g.resolverIdx] = struct{}{}
 		resolver := g.resolvers[g.resolverIdx]
-		if resolver.IsExhausted() {
```

Comment:
Per in-person discussion, `IsExhausted` is no longer useful since the removal of `"lb"` (load balancer) resolvers.